ctapipe/resources/train_disp_reconstructor.yaml

```diff
@@ -4,25 +4,50 @@
 #
 # Configuration for training machine-learning models
 # ======================================================================
+
 TrainDispReconstructor:
+  random_seed: 0              # Seed used for sampling n_events for training.
+  n_events:                   # The number of events used for training that can be provided
+    # - [type, "LST*", 1000]  # independently for each telescope type (e.g. "LST_LST_LSTCam").
+    # - [type, "MST*", 1000]  # If not specified, all events in the file are used.
+
   CrossValidator:
     n_cross_validations: 5
+    rng_seed: 1337
 
   DispReconstructor:
-    norm_cls: ExtraTreesRegressor
-    norm_config:
+    prefix: disp  # Prefix of the output generated by the DispReconstructor (Default: "disp")
+
+    # All regression algorithms in scikit-learn are supported (https://scikit-learn.org/stable/modules/classes.html)
+    norm_cls: ExtraTreesRegressor  # scikit-learn algorithm to be used for the norm(disp) models
+    norm_config:                   # Settings passed to the scikit-learn algorithm
       n_estimators: 10
       max_depth: 10
       n_jobs: -1
 
-    log_target: True
+    log_target: True  # If true, norm(disp) models predict log(norm(disp)) (output = exp(prediciton))
```

Comment:
I don't think it's a good idea to document the options in the config files, they are available in the documentation:
https://ctapipe.readthedocs.io/en/latest/api/ctapipe.reco.DispReconstructor.html#ctapipe.reco.DispReconstructor.log_target

Duplicating this by hand has the danger of getting out of sync.

Reply:
@LukasBeiske I think Max has a good point that we shouldn't duplicate the existing documentation in the example config files.

Instead I suggest you open a PR where you update the `DispReconstructor` with the doc strings you suggest here, as I find them more useful.

Reply:
Ok, I can see that. This extends to all the config options of the ML Reconstructor classes and the stereo combiner options, as well, doesn't it?

I noticed that there are some docstrings missing in `DispReconstructor` and the other ML related classes, so I will put all of that in a separate PR :+1:

Reply:
In that case, should I also remove these options for which I am only setting the default values here (like e.g. `prefix`, and `StereoMeanCombiner.weights`)?

Reply:
See #2456

Reply:
> In that case, should I also remove these options for which I am only setting the default values here (like e.g. `prefix`, and `StereoMeanCombiner.weights`)?

I think for the example config it makes more sense to change `disp` to something custom (maybe "elite" ; )  as an example to the reader. The stereo combiner weights is something you choose for more principled reasons so maybe remove it.

Reply:
I don't think we should change the prefixes by default. For most purposes, we should stick to (and properly define them) the default prefixes.

Otherwise, we'll end up with different prefixes / data formats for the same things done by different users. (or all of them using the same one we now defined as "new default" in the config file.

I'd say comment out the "prefix" completely and add a comment like "add a prefix here if you e.g. want to compare the same method applied with different settings".

Reply:
> I think for the example config it makes more sense to change `disp` to something custom (maybe "elite" ; ) as an example to the reader. The stereo combiner weights is something you choose for more principled reasons so maybe remove it.

I removed the default value for the weights and commented it out, just like `prefix`. I think leaving it in as a comment like this makes it a bit more obvious how one can set the options of `StereoMeanCombiner` or any other future stereo combiner.